content/en/tracing/trace_collection/compatibility/go.md

```diff
@@ -207,33 +270,50 @@ The Go tracer includes support for the following data stores and libraries.
 | [AWS SDK][20]           | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/aws/aws-sdk-go/aws][21]                |
 | [AWS SDK v2][75]        | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/aws/aws-sdk-go-v2/aws][76]                |
 | [Elasticsearch][22]     | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/olivere/elastic][23]                   |
+| [Elasticsearch v6][85] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/elastic/go-elasticsearch.v6][86]     |
 | [Cassandra][24]         | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gocql/gocql][25]                       |
-| [GraphQL][26]           | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/graph-gophers/graphql-go][27]          |
+| [GraphQL (graph-gophers)][26]           | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/graph-gophers/graphql-go][27]          |
+| [GraphQL (graphql-go)][87]           | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/graphql-go/graphql][88]          |
+| [GraphQL (gqlgen)][89] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/99designs/gqlgen][90]                 |
 | [HTTP][28]              | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http][29]                          |
 | [HTTP router][30]       | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/julienschmidt/httprouter][31]          |
 | [Redis (go-redis)][32]  | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis][33]                    |
-| [Redis (go-redis-v8)][34]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis.v8][35]                |
+| [Redis (go-redis-v7)][32]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis.v7][91]                |
+| [Redis (go-redis-v8)][32]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis.v8][35]                |
+| [Redis (go-redis-v9)][32]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/redis/go-redis.v9][92]                |
 | [Redis (redigo)][36]    | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/garyburd/redigo][37]                   |
 | [Redis (new redigo)][38]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/gomodule/redigo][39]                   |
+| [Redis (rueidis)][93]| Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/redis/rueidis][94]                   |
 | [SQL][40]               | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql][41]                      |
 | [SQLx][42]              | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/jmoiron/sqlx][43]                      |
+| [PostgreSQL (pgx v5)][95] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/jackc/pgx.v5/v2][96]                  |
```

Comment:
```suggestion
| [PostgreSQL (pgx v5)][95] | Fully Supported | [gopkg.in/DataDog/dd-trace-go.v1/contrib/jackc/pgx.v5][96]                  |
```

Reply:
It looks like this is the expected link text, but please confirm and update if needed.